### ISSUE TITLE
process.wait(0) when traing faster_rcnn

### ISSUE THREAD

**Issue by kangkang59812** [pytorch/vision#1643]
<img width="575" alt="662FE0EB-CD0B-4FED-B036-CB5964218D31" src="https://user-images.githubusercontent.com/12228641/70307513-51f3db00-1844-11ea-98ae-a814d527d4b1.png">
the program always stops at epoch 27(just stop, no more output), and when I use `ctrl+c` to interrupt the program, the content is as shown in the pic.

**Comment by fmassa** [comment]
Hi,

This is weird. I'm not sure if you are facing a deadlock (which could be due to several reasons), or if something else is happening.

When your program stopped, were the GPUs running at 100% utilization?

**Comment by kangkang59812** [comment]
@fmassa yes, 100%. I didn't change the setting about thread in train.py, just making a custom dataset using h5py.

**Comment by fmassa** [comment]
This makes me think that you should check if your install of `h5py` is thread-safe.
If not, then it might cause issues with dataloading.

Also, if that's not the case, then debugging a CUDA deadlock is very annoying. Some information about it can be found in https://github.com/pytorch/vision/issues/1148

Just to cross this out, does the code deadlock if running on the standard COCO dataset?
